workshops/splatter_paint/es-xl.md

```diff
@@ -0,0 +1,191 @@
+
+```js
+var path = new Path.Circle({
+  center: event.middlePoint,
+  radius: 10
+})
+```
+
+Si ejecutas tu repl ahora y mueves el mouse, verás ... nada. De hecho, estás dibujando círculos, pero los círculos son actualmente transparentes. Entonces, démosles un poco de color. En el fragmento de código anterior, aún en la función `onMouseMove` , añade:
+
+```js
+path.fillColor = {
+  hue: 0,
+  saturation: 1,
+  brightness: 1
+}
+```
+
+Ahora mismo, todo tu archivo `script.js`  debería verse así:
+
+```js
+function onMouseMove(event) {
+  var path = new Path.Circle({
+    center: event.middlePoint,
+    radius: 10
+  })
+  path.fillColor = {
+    hue: 0,
+    saturation: 1,
+    brightness: 1
+  }
+}
+```
+
+En lugar de usar los sistemas de color hexadecimal o RGB más comunes, Paper.js usa el sistema de color HSB, que usa ángulos en una rueda de color para describir el color. En el sistema de color HSB, 0 = 0° = rojo, y `360*n`° también es rojo.
+
+![Rueda de color en la que a cada color se le asigna un valor de grado](img/hsb-color-wheel.PNG)
+
+(Si estás interesado en obtener más información sobre el sistema de color HSB, consulta [esta fantástica explicación](https://learnui.design/blog/the-hsb-color-system-practicioners-primer.html))
+
+Con esto en mente, intenta ejecutar tu repl ahora. Porque pusiste el `hue` en 0, ¡ahora estás dibujando círculos rojos! Pero tus círculos solo aparecen en la esquina superior izquierda de la pantalla.Esto se debe a que el ancho y la altura de su lienzo están configurados actualmente en los valores predeterminados de Paper.js. (300px por 150px).
+
+Hagamos que tu lienzo ocupe toda la pantalla. En tu archivo `style.css` , agrega lo siguiente:
+
+```css
+canvas {
+  width: 100%;
+  height: 100%;
+}
+```
+
+Corre tu repl de nuevo.
+
+Casi llegamos. El CSS que acabas de escribir establece el ancho y la altura del lienzo al 100% del elemento principal. En tu archivo `index.html` , `<body>` es el elemento padre del `<canvas>` porque creaste el lienzo entre las etiquetas `<body>`.
+`<body>` es actualmente tan grande como su propio elemento padre, `<html>`, que no ocupa la pantalla del todo. Con esto en mente, agrega lo siguiente a tu archivo CSS:
+
+```css
+html,
+body {
+  width: 100%;
+  height: 100%;
+  margin: 0;
+}
+```
+
+Esto establece el ancho y alto del cuerpo al ancho y alto de tu pantalla y elimina cualquier espacio adicional entre el borde de la pantalla y el cuerpo.
+
+Solo para recapitular: todo tu archivo CSS ahora debería verse así:
+
+```css
+html,
+body {
+  width: 100%;
+  height: 100%;
+  margin: 0;
+}
+canvas {
+  width: 100%;
+  height: 100%;
+}
+```
+
+Si corres tu repl de nuevo, Deberías notar que tus círculos rojos ahora están llenando toda la pantalla. Woohoo!
+
+![Patrón aleatorio formado por círculos rojos](https://cloud-3aosybiuc-hack-club-bot.vercel.app/6red-circles.jpg)
+
+## Haciendo salpicaduras
+
+Estamos llegando a alguna parte, pero esto todavía no se siente muy salpicado.
```

Comment:
I know that's the literal translation, but "pero esto no se siente muy salpicado" sounds weird in Spanish
```suggestion
Estamos llegando a alguna parte, pero esto todavía no se ve muy salpicado.
```